lakey_service/downloader/executors/databricks.py

```diff
@@ -1,13 +1,43 @@
+from databricks_api import DatabricksAPI
+from datetime import datetime
+from django.conf import settings
 
+from .baseexecutor import BaseExecutor
 
-class DataBricksExecutor:
-    # https://docs.databricks.com/api/latest/jobs.html#jobsnotebooktask
 
-    def start_cluster(self):
-        pass
+class DatabricksExecutor(BaseExecutor):
 
-    def start_job(self, spec):
-        pass
+    def __init__(self):
+        self.db = DatabricksAPI(
+            host=settings.DATABRICKS_HOST,
+            token=settings.DATABRICKS_TOKEN
+        )
 
-    def get_job_status(self, job_id):
-        pass
+    def __generate_file_path(self):
+        """ Return path of output file """
+        now = datetime.now()
```

Comment:
please use 
```
from django.utils import timezone

now = timezone.now()
```

This is timezone aware version of the datetime. The timezone is taken from django settings file.